packages/rum/src/domain/record/serializationUtils.ts

```diff
@@ -58,6 +64,9 @@ export function makeSrcsetUrlsAbsolute(attributeValue: string, baseUrl: string)
 
 export function makeUrlAbsolute(url: string, baseUrl: string): string {
   try {
+    if (isExperimentalFeatureEnabled('base-tag')) {
+      return url.trim()
```

Comment:
Do you need the `.trim()` here? I'd suggest removing it.

Reply:
I was pondering that myself. Why did we trim in the first place?